plugins/decklink/audio-repack.h

```diff
@@ -26,11 +26,10 @@ struct audio_repack {
 };
 
 enum _audio_repack_mode {
-	repack_mode_8to3ch_swap23,
-	repack_mode_8to4ch_swap23,
-	repack_mode_8to5ch_swap23,
-	repack_mode_8to6ch_swap23,
-	repack_mode_8ch_swap23_swap46_swap57,
+	repack_mode_8to3ch=3,
+	repack_mode_8to4ch,
+	repack_mode_8to5ch,
+	repack_mode_8to6ch,
```

Comment:
There's a trailing comma after the last enum element.  I don't think it actually matters either way, but you could remove it since the line has changed.